app/helpers/claims_helper.rb

```diff
@@ -73,4 +73,12 @@ def subject_list(subjects)
       two_words_connector: connector
     )
   end
+
+  def claim_school_title(searching_for_additional_school)
+    searching_for_additional_school ? t("student_loans.questions.additional_school") : t("student_loans.questions.claim_school")
+  end
+
+  def school_search_value(searching_for_additional_school, current_claim)
```

Comment:
Maybe there's a more intention-revealing name for this helper? how about `school_search_prefill_value`?

Reply:
Yeah, you're right on that. This came as part of a refactor, and I may have got a little helper-happy